jaggery/requirements.in

```diff
@@ -13,4 +13,5 @@ ipython==8.12.3
 djangorestframework==3.15.1
 celery==5.4.0
 black==24.4.2
-flake8==7.0.0
\ No newline at end of file
+flake8==7.0.0
+django-celery-results==2.5.1
```

Comment:
Lets see if we can use a different store for djang-celery-results instead of the database
@TODO